documents-storage-jcr/src/main/java/org/exoplatform/documents/storage/jcr/JCRDocumentFileStorage.java

```diff
@@ -631,15 +631,16 @@ public void renameDocument(long ownerId, String documentID, String title, Identi
       }
       String name = Text.escapeIllegalJcrChars(cleanName(title));
       //clean node name
-
       name = URLDecoder.decode(name,"UTF-8");
-
-      String oldName = node.getName();
-      if (oldName.indexOf('.') != -1 && node.isNodeType(NodeTypeConstants.NT_FILE)) {
+      if(name.indexOf('.') == -1){
+      String oldName = node.getName().indexOf('.') == -1 && node.isNodeType(NodeTypeConstants.NT_FILE)
+                           && node.hasProperty(NodeTypeConstants.EXO_TITLE) ? node.getProperty(NodeTypeConstants.EXO_TITLE).getString(): node.getName();
+       if (oldName.indexOf('.') != -1 && node.isNodeType(NodeTypeConstants.NT_FILE)) {
```

Comment:
fix indentation

Reply:
indentation problem is not fixed

Reply:
fixed

Reply:
https://github.com/exoplatform/documents/blob/d56e663a4f7de2430b4899aadd0ae3fd97b49372/documents-storage-jcr/src/main/java/org/exoplatform/documents/storage/jcr/JCRDocumentFileStorage.java#L579-L583

still not good